scripts/mark-definite-framework-results.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+"""
+Uses a number of supplied json schemas to determine whether suppliers on a particular framework should be marked as
```

Comment:
Oh except the subschema bit - all that happened there was that I designed in a graceful behaviour to follow if the subschema is not found by the script in the superschema. It's not a big deal, and no, none of our current frameworks do things this way. It would correspond to a framework where all of the questions were mandatory.